src/main/java/bot/telegram/HttpClientResponse.java

```
@@ -0,0 +1,73 @@
+package bot.telegram;
+
+import org.json.JSONObject;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class HttpClientResponse {
+    Integer statusCode;
+    List<String> headers;
+    String body;
+
+    public HttpClientResponse(Integer statusCode, List<String> headers, String body) {
+        this.statusCode = statusCode;
+        this.headers = headers;
+        this.body = body;
+
+        /*System.out.println("Status code: " + statusCode);
+        System.out.println("Headers: " + headers);
+        System.out.println("Body: " + body);*/
+    }
+
+    public String getScoreValue(Boolean localContentTest){
+        String jsonPathName;
+        String content;
+
+        jsonPathName = "./src/main/resources/scannertest.json";
+
+        if(localContentTest) {
+            try {
+                content = new String(Files.readAllBytes(Paths.get(jsonPathName)));
+            } catch (FileNotFoundException e) {
+                throw new RuntimeException(e);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
```

Well done! Here, you’ve chosen not to silence the exception but instead rethrow it, which is a good practice for maintaining clarity and propagating errors appropriately.